incubator/zookeeper/README.md

```diff
@@ -148,7 +174,8 @@ $ kubectl exec <RELEASE-NAME>-2 -- /opt/zookeeper/bin/zkCli.sh get /foo;
 
 Watch existing members:
 ```console
-$ kubectl run --attach bbox --image=busybox --restart=Never -- sh -c 'while true; do for i in 0 1 2; do echo zk-$i $(echo stats | nc <pod-name>.<headless-service-name>:2181 | grep Mode); sleep 1; done; done';
+$ 
+ 'while true; do for i in 0 1 2; do echo zk-$i $(echo stats | nc <pod-name>.<headless-service-name>:2181 | grep Mode); sleep 1; done; done';
```

Comment:
Why do we no longer need the busybox? We still need to execute this from within the cluster IIUC.
Maybe we should break up the `kubectl run` into its own step for clarity though.